subscriptions.go

```diff
@@ -191,6 +192,9 @@ type UpdateSubscription struct {
 	NetTerms               NullInt              `xml:"net_terms,omitempty"`
 	PONumber               string               `xml:"po_number,omitempty"`
 	SubscriptionAddOns     *[]SubscriptionAddOn `xml:"subscription_add_ons>subscription_add_on,omitempty"`
+	GatewayCode            string               `xml:"gateway_code,omitempty"`
```

Comment:
I don't see `gateway_code` here:
https://dev.recurly.com/docs/update-subscription

But oddly enough I do see it here:
https://dev.recurly.com/docs/update-subscription-with-add-ons

Did you try this on the "Update Subscription" call and it's just not documented? Or should this just be part of the "Update Subscription Notes" flow?

Reply:
So you can update this field on three documented API items:
https://dev.recurly.com/docs/update-subscription-with-add-ons
https://dev.recurly.com/docs/update-subscription-with-custom-fields (same API as above)
https://dev.recurly.com/docs/update-subscription-notes

I'd rather put it in the normal update subscription interface, as the encoded payload is actually the same as normal subscription update.